documentation/meta/media_properties/etls.md

```diff
@@ -0,0 +1,303 @@
+there are also transformations that occur when constructing the ES documents.
+The
+[exhaustive list can be seen in the `elasticsearch_models` module](https://github.com/WordPress/openverse/tree/main/ingestion_server/ingestion_server/elasticsearch_models.py),
+which includes:
+
+#### Common
+
+- Computing the authority boost
+- Computing the popularity
+- Parsing a description from the media, if available
+- Parsing tags
+- Determining the `mature` field
+
+#### Images
+
+- Computing the aspect ratio
+- Computing the size
+
+#### Audio
+
+- Computing the duration
+
+### Dead link filtering
+
+When records are retrieved from Elasticsearch, they also go through a process
+called "dead link filtering". This operation attempts to remove links to media
+which the API may have trouble accessing in order to prevent dead or invalid
+links from being surfaced in API results. This process both reads from and
+writes to Redis, where information about valid/invalid URLs is cached.
+
+The full set of dead link logic can be found in the
+[`check_dead_link` module](https://github.com/WordPress/openverse/tree/main/api/api/utils/check_dead_links/__init__.py).
+
+### API serializers
+
+An additional transformation occurs after a record is retrieved from the
+database (and once it has been validated by the dead link filtering). This step
+includes any transformations that required in hydrating the database record and
+converting it into a JSON object which is sent to the requester.
+
+Serializations can be found
+[in the codebase under `api/api/serializers`](https://github.com/WordPress/openverse/tree/main/api/api/serializers).
+
+### Attribution generation
+
+The frontend also modifies results before displaying them to a user. This
+includes generating attributions in multiple formats. The full attribution
+generation code
+[can be found in `attribution-html.ts`](https://github.com/WordPress/openverse/tree/main/frontend/src/utils/attribution-html.ts).
+
+## Proposed
+
+The projects #430, #431, and #3925 all intent to modify the above process. Below
+is the diagram of what this process might look like after all steps are taken.
+Most blocks reference the same sections above, with the exception being
+[Deleted media & tag filtering](#deleted-media--tag-filtering).
+
+```{mermaid}
+flowchart TD
+
+%% DBs
+A[(API DB)]
+ES[(Elasticsearch)]
+
+%% Graph definitions
+    subgraph AF [Airflow]
+    %% ETLs
+    M(MediaStore):::ETL
+    IC1(Index constraints):::ETL
+    B(Batched update):::ETL
+    ING(Deleted media & tag filtering):::ETL
+    IC2(Index constraints):::ETL
+    ESF(Index creation & deleted media filtering):::ETL
+
+    P{{Provider}} --> M
+    M --> T{{TSV}}
+    T --> IC1
+    IC1 --> C[(Catalog DB)]
+    C --> B --> C
+    C --> ING
+    end
+
+    %% Connections for the data refresh
+    ESF --> ES
+    ING --> A
+    A --> IC2 --> A
+    A --> ESF
+
+    subgraph DJA [DjangoAPI]
```

Comment:
```suggestion
    subgraph DJA [Django API]
```

Reply:
For some reason I remember this causing an error...but I just checked and it's fine! I'll fix the other reference too 🚀